test/views/behaviours/ViewBehaviours.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.behaviours
+
+import base.SpecBase
+import org.jsoup.nodes.Document
+
+trait ViewBehaviours extends SpecBase {
```

Comment:
There is already a `ViewSpecAssertions` should these go there?

Reply:
Rather than trait stacking, we can just make this a `object` and import the methods. What do you think? Trait stacking has a few complication, for example `SingleViewSpec` brings in `SpecBase` already, I don't think this will be a problem here but it's just an example of how trait stacking is limited. We can use  `objects` + `import`s as an alternative (ie object composition)

Reply:
Agreed. Have gone a slightly different way and made sure the stacking is linear rather than bringing in the same traits twice.